modules/containers-signature-verify-application.adoc

```diff
@@ -0,0 +1,251 @@
+// Module included in the following assemblies:
+//
+// * security/container_security/security-container-signature.adoc
+
+[id="containers-signature-verify-application_{context}"]
+= Verifying the signature verification configuration
+After applying machine configs to the cluster, the machine-config-controller notices the new Machine Config and generate a new version of the worker, such as `rendered-worker-<hash>`.
+
+.Prerequisites
+* You have enabled signature verification using a machine config file
+
+.Procedure
+
+. The following command displays all information about that selected worker.
```

Comment:
s/The following command displays/Run the following command to display
Maybe get rid of _all_? _display information about_

Reply:
s/./:
For procedural steps